src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,13 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machine = { new Excavator(), new Bulldozer(), new Truck() };
+        int count = 0;
+        while (machine.length > count) {
+            machine[count].doWork();
+            machine[count].stopWork();
+            count++;
```

Comment:
Just use for each loop instead of this

Reply:
I don't understand what i should to do, what i should change

Reply:
use "for", not a while?